account_bank_statement_import_paypal/README.rst

```diff
@@ -46,17 +46,17 @@ Usage
 
 To import statements from PayPal via Statement report:
 
-#. Go to `PayPal <https://paypal.com/>`__
-#. Open *Reports > Statements* and select *Monthly* or *Custom*
+#. Go to `PayPal <https://business.paypal.com/merchantdata/reportHome/>`__
```

Comment:
This seems to be an improvement that would be meaningful to cherry-pick in v12, please separate

Reply:
What do you mean? These are updated instructions that my colleague found when trying to get the statement.

Reply:
I mean that they would be useful (updated instructions) on v12 as well? And that should be as separate commit?

Reply:
I will update the instructions in v12.

Reply:
Extracted in c8b233a0d0e632244fa3166407e57152c9b15e20